tests/resources/Nimbus-Util.robot

```diff
@@ -13,9 +13,14 @@ Deploy Nimbus ESXi Server
     Open Connection  %{NIMBUS_GW}
     Login  ${user}  ${password}
 
-    ${out}=  Execute Command  nimbus-esxdeploy ${name} --disk=48000000 --ssd=24000000 --memory=8192 --nics 2 ${version}
-    # Make sure the deploy actually worked
-    Should Contain  ${out}  To manage this VM use
+    :FOR  ${IDX}  IN RANGE  1  5
+    \   ${out}=  Execute Command  nimbus-esxdeploy ${name} --disk=48000000 --ssd=24000000 --memory=8192 --nics 2 ${version}
+    \   # Make sure the deploy actually worked
+    \   ${status}=  Run Keyword And Return Status  Should Contain  ${out}  To manage this VM use
+    \   Exit For Loop If  ${status}
+    \   Log To Console  Loop ${IDX} failed, trying again in 5 minutes
```

Comment:
Should this say "Loop %s failed..." or maybe "Nimbus deployment %s failed..." I know this is a minor nitpick, but it might save some confusion at some point.